### PR TITLE
Us#03 task#12

### PR DESCRIPTION
**Pull request by sarthak-tiwari**
The file QuestionAttemptTemplate is the file to be used by other people.

The file QuestionStub and TestRunner are to show a running example of how to use QuestionAttemptTemplate.

All the functionality has been implemented and hopefully only UI changes (if any) will be coming in the future.

### CODE REVIEW

**Review by rubyqzhao — APPROVED** [review]
Hi Sarthak - This looks good for some initial functionality. I'm approving the changes, but you might also want to ask some of the other people working on the student side to review your changes.

File: src/view/TestRunner.java [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+ * Form to show a question and navaigation buttion and to give user
+ * functionality to select one of the options.
+ * @author      Sarthak Tiwari <sarthak.tiwari@asu.edu>
+ * @version     1.0
+ */
+public class TestRunner {
+
+    public static void main(String[] args) {
+
+        JFrame f = new JFrame();
```
Comment (sverma23x) [inline src/view/TestRunner.java:16]: let's replace 'f' with something meaningful.

File: src/view/QuestionAttemptTemplate.java [hunk 2]
```diff
@@ -0,0 +1,282 @@
...
+    private static JRadioButton option2;
+    private static JRadioButton option3;
+    private static JRadioButton option4;
+    private static ButtonGroup options;
+
+    private static JButton next;
+    private static JButton giveup;
+
+    private static short correctAnswer = 0;
+    private static int isCorrectAnswer;
```
Comment (sverma23x) [inline src/view/QuestionAttemptTemplate.java]: variable name starting with 'is' like in this case, 'isCorrectAnswer' should be boolean not int.

**Review by sverma23x — CHANGES_REQUESTED** [review]
Added a few comments. Please resolve them.

File: src/view/TestRunner.java (hunk 1 above)
Comment (sarthak-tiwari) [reply src/view/TestRunner.java:16]: Test Runner class won't be part of the deliverable, that is just a sample code showing how to use the class.

File: src/view/QuestionAttemptTemplate.java (hunk 2 above)
Comment (sarthak-tiwari) [reply src/view/QuestionAttemptTemplate.java]: Umm.. it still represents whether the answer is correct or not.. can you suggest some name ?

File: src/view/TestRunner.java (hunk 1 above)
Comment (sarthak-tiwari) [reply src/view/TestRunner.java]: Does 'reponseStatus' sound okay ?

File: src/view/QuestionAttemptTemplate.java (hunk 2 above)
Comment (sarthak-tiwari) [reply src/view/QuestionAttemptTemplate.java]: Does 'reponseStatus' sound okay ?

File: src/view/TestRunner.java (hunk 1 above)
Comment (sverma23x) [reply src/view/TestRunner.java:16]: ok

File: src/view/QuestionAttemptTemplate.java (hunk 2 above)
Comment (sverma23x) [reply src/view/QuestionAttemptTemplate.java]: yeah ok. Sounds fine.

**Review by sverma23x — COMMENTED** [review]
Looks good. Approving changes.

**Review by sverma23x — APPROVED** [review]

**Pull request merged**